src/components/Swap/Toolbar/ToolbarOrderRouting.test.tsx

```diff
@@ -0,0 +1,65 @@
+import { CurrencyAmount, TradeType } from '@uniswap/sdk-core'
+import { InterfaceTrade } from 'state/routing/types'
+import { renderComponent } from 'test'
+import { buildMultiV3Route, buildSingleV3Route, DAI, USDC } from 'test/utils'
+
+import ToolbarOrderRouting from './ToolbarOrderRouting'
+
+describe('ToolbarOrderRouting', () => {
+  it('renders correctly, loading', () => {
+    const component = renderComponent(<ToolbarOrderRouting />)
+    expect(component.getByText('Order routing')).toBeTruthy()
```

Comment:
These tests don't check anything but the title. Is that intentional?
Would this be suited for a snapshot test?